index.js

```diff
@@ -13,6 +13,7 @@ function expressPlugin (fastify, options, next) {
   fastify.use = use
   fastify[kMiddlewares] = []
   fastify[kExpress] = Express()
```

Comment:
I think we should expose the main express instance.

Reply:
instead of calling `disable` here you mean? Would I do that by just `fastify.expressApp = Express()` or something else?

Reply:
I would just do `fastify.decorate('express', Express())` and remove the symbol.

I kind of prefer to keep express as vanilla as possible to avoid support work.

Reply:
@mcollina I pushed that now, thoughts?

Reply:
#8